src/NuGet.Core/NuGet.ProjectModel/PackageSpecUtility.cs

```diff
@@ -101,9 +114,23 @@ public static bool IsSnapshotVersion(string version)
                 throw new ArgumentNullException(nameof(projectFramework));
             }
 
+            var hasImports = imports?.Any() == true;
             var hasATF = assetTargetFallback?.Any() == true;
             var hasPTF = packageTargetFallback?.Any() == true;
 
+            // Legacy Project.Json case
+            if (hasImports && !hasATF && !hasPTF)
+            {
+                if (isAssetTargetFallback)
```

Comment:
leftover bad comment...the comments needs moved to the line where isAssetTargeTFallback is false, because that's the Project.Json case....the case where we have imports and isAssetTargetFallback is true is when we are reading an assets file generated by the CLI (which contains the code prior to Justin's change)